interface/FrankWriter.h

```diff
@@ -100,14 +105,22 @@ template <class EventClass> class FrankWriter : public BaseOperator<EventClass>
 
           // clear jet collection
           mix_jets_.clear();
-
+	  best_match_distances_.clear();
```

Comment:
basically you have to do as is done for ev.best_match_hemispheres but doing ev.best_match_distances_.emplace_back(knn_), remember also to clear it at the beginning as is done for
https://github.com/pablodecm/amva4np_analysis/pull/1/files#diff-9caf637451561fc971ed67803ad972b9R243